File: 9-112A-Petya&Strings/tes9.cpp
#include <iostream>
using namespace std;

int charcase(string a, string b){

    for (int i = 0; i<a.length(); i++ ) {
        char A = tolower(a[i]);
        char B = tolower(b[i]);
        
        if (A > B ) {
            return 1;
        }
        else if (A < B) {
            return -1;    
        }
    }
    
    return 0;
}

int main()
{
    string a,b;
    
    cin>>a>>b;

    cout<<charcase(a,b);
    
    return 0;
}

File: 9-112A-PetyaANDStrings/tes9.go
package main

import (
	"fmt"
	"strings"
)

func charcase(a string, b string) int {
	for i := 0; i < len(a); i++ {
		A := strings.ToLower(string(a[i]))
		B := strings.ToLower(string(b[i]))

		if A > B {
			return 1
		} else if A < B {
			return -1
		}
	}

	return 0
}

func main() {
	var a, b string

	fmt.Scan(&a, &b)

	result := charcase(a, b)

	fmt.Println(result)
}
